### PR TITLE
Add Visual Studio 2017 compatibility. Fixes #17

### LINKED ISSUE #17: Visual Studio 2017 support?
Any chance of updating this extension to support Visual Studio 2017?  I'm on day one of using VS 2017 and I already miss VSFileNav like crazy!

I tried using the existing installer from https://marketplace.visualstudio.com/items?itemName=IPWright83.VSFileNav-17743, but the VSIX installer reported "This extension is already installed to all applicable products."

Thanks!  (Let me know if I can do anything to help!)

### PR DESCRIPTION
**Pull request by jschneid**
- Change target framework to 4.6 (from 4.5.2) to fix an error about
reference compatibility (http://stackoverflow.com/a/34299520/12484)
- Change Visual Studio version from 14 (VS2015) to 15 (VS2017)
- Increment extension version to 15.2 (from 15.1)

### CODE REVIEW

File: VSNav/source.extension.vsixmanifest [hunk 1]
```diff
@@ -31,4 +31,7 @@ For more information please visit https://github.com/IPWright83/VSFileNav
   <Assets>
     <Asset Type="Microsoft.VisualStudio.VsPackage" d:Source="Project" d:ProjectName="%CurrentProject%" Path="|%CurrentProject%;PkgdefProjectOutputGroup|" />
   </Assets>
+  <Prerequisites>
+    <Prerequisite Id="Microsoft.VisualStudio.Component.CoreEditor" Version="[15.0.26208.0,16.0)" DisplayName="Visual Studio core editor" />
```
Comment (jschneid) [inline VSNav/source.extension.vsixmanifest:35]: This is a new requirement, per https://docs.microsoft.com/en-us/visualstudio/extensibility/how-to-migrate-extensibility-projects-to-visual-studio-2017

**Comment by jschneid** [comment]
I tested installing the newly-built VSNav.vsix from this branch on my local Visual Studio 2017, and it seemed to work great!

I didn't try installing the new .vsix on any older Visual Studio versions, though.

**Comment by IPWright83** [comment]
@jschneid thanks very much. I'll do some quick testing of it over the rest of this week. If all's good I'll merge in the PR :)

**Comment by zack-snyder** [comment]
What is the current state here? Is it VS 2017 read?

**Comment by jschneid** [comment]
FWIW, I've been using the build from this PR since March with my local copy of VS 2017 with no issues!

**Comment by ghost** [comment]
@IPWright83, @jschneid:
This is not working for me. It seems it still installs version 15.1 instead of 15.2, that might be the problem.

Also, it'd be nice to have the new version uploaded to the Visual Studio Marketplace.

**Pull request merged**